### DISCUSSION TITLE
Having a TQDM bar would be awsome to folow the "mp_compute_indicator".

### DISCUSSION THREAD

**Discussion by mreismendes** [Boulder-Investment-Technologies/lppls#38] [Ideas]
What do you think about it?

PS: I'm not a software engineer. I'm just a data scientist that sometimes goes deep into under the hood issues.

Thank you!

**Comment by Joshwani** [comment]
Yeah, that would be nice. Especially since some of the large scale indicators can take forever. I can look into tqdm.

**Comment by Joshwani** [comment]
This has been merged https://github.com/Boulder-Investment-Technologies/lppls/pull/66
